src/test/java/io/nats/client/utils/TestBase.java

```diff
@@ -120,10 +120,18 @@ public static void runInJsServer(ErrorListener el, InServerTest inServerTest) th
         runInServer(false, true, new Options.Builder().errorListener(el), null, inServerTest);
     }
 
+    public static void runInJsServer(VersionCheck vc,  ErrorListener el, InServerTest inServerTest) throws Exception {
+        runInServer(false, true, new Options.Builder().errorListener(el), vc, inServerTest);
+    }
+
```

Comment:
macro to make it easier to test only if a server version is appropriate. Useful during regression tests against older servers.